CMakeLists.txt

```diff
@@ -149,19 +149,28 @@ endfunction()
 add_subdirectory(book)
 add_subdirectory(examples)
 
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Gui Test REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Test REQUIRED)
+get_target_property(QMAKE Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
+set(CARGO_ENV "QMAKE=${QMAKE};CARGO_TARGET_DIR=${CMAKE_CURRENT_SOURCE_DIR}/target")
+
 # Create helper method which adds relevent cargo tests for a given manifest
 function(add_test_cargo TEST_NAME_PREFIX MANIFEST_PATH ADD_DOCTESTS)
```

Comment:
can we add a `add_test_cargo` line for `qt-build` like we have for the other Rust crates, so that we ensure that fmt/clippy/tests are run.